src/test/java/com/commercetools/sync/products/utils/ProductUpdateActionUtilsTest.java

```diff
@@ -88,13 +96,35 @@ public void buildVariantsUpdateActions_updatesVariants() {
             .containsExactlyInAnyOrder(RemoveVariant.ofVariantId(2, true), RemoveVariant.ofVariantId(3, true));
 
         // check add actions
-        ProductVariantDraft draftMaster = productDraftNew.getMasterVariant();
-        ProductVariantDraft draft5 = productDraftNew.getVariants().get(1);
-        ProductVariantDraft draft6 = productDraftNew.getVariants().get(2);
+
+        final ProductVariantDraft draftMaster = productDraftNew.getMasterVariant();
+        final ProductVariantDraft draft5 = productDraftNew.getVariants().get(1);
+        final ProductVariantDraft draft6 = productDraftNew.getVariants().get(2);
+        final ProductVariantDraft draft7 = productDraftNew.getVariants().get(3);
+
         assertThat(updateActions).contains(
-            buildAddVariantUpdateActionFromDraft(draftMaster),
-            buildAddVariantUpdateActionFromDraft(draft5),
-            buildAddVariantUpdateActionFromDraft(draft6));
+            AddVariant.of(draftMaster.getAttributes(), draftMaster.getPrices(), draftMaster.getSku(), true)
```

Comment:
Why this change? I don't see any assert with Assets here.

Reply:
this is not related to assets. I changed to actually asserting if the correct actions are actually build  instead of using a method that is **under test** (`buildAddVariantUpdateActionFromDraft`) in the **test**..